projects/plugins/jetpack/extensions/blocks/dialogue/save.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { BASE_CLASS_NAME } from './utils';
+import { convertTimeCodeToSeconds } from '../../shared/components/media-player-control/utils';
+
+export default function save( { attributes } ) {
+	const { content, participantLabel, showTimestamp, timestamp } = attributes;
+
+	return (
+		<div>
+			<div className={ `${ BASE_CLASS_NAME }__meta` }>
+				<div className={ `${ BASE_CLASS_NAME }__participant` }>
+					{ participantLabel }
+				</div>
+				{ showTimestamp && (
+					<div className={ `${ BASE_CLASS_NAME }__timestamp` }>
+						<a
+							className={ `${ BASE_CLASS_NAME }__timestamp_link` }
+							href={ `#${ convertTimeCodeToSeconds( timestamp ) }` }
```

Comment:
we store the timestamp in the URL hash instead of a `data` attribute because some envs filter these attributes. Handled here 5ddf0cac868de1137e0372a7f6f6c84b510ebcef and here 0cd04ee4738311d2640b34c1760a1c5d67b44503